.github/workflows/ci.yml

```
@@ -9,6 +9,7 @@ env:
 
 jobs:
   test:
+    name: build and test
```

I personally think the name attribute is redundant.

```suggestion
build-and-test: 
```

Same comment for the other jobs below.